Documentation/reference/config.md

```diff
@@ -59,7 +75,8 @@ notifier:
     webhook: null
     amqp: null
     stomp: null
-auth: {}
+auth: 
+  psk: {}
```

Comment:
```suggestion
  psk: null
```

Reply:
I'm using {} to signify a further object exists there